.gitignore

```diff
@@ -9,6 +9,6 @@ jest-preset.json
 # misc
 .DS_Store
 npm-debug.log
-
+*.tgz
```

Comment:
probably better to keep the newline between sections convention seen in the other sections
```suggestion
*.tgz

```